src/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -0,0 +1,131 @@
+# declaration-block-no-redundant-longhand-properties
+
+Disallow longhand properties that can be combined into one.
+
+```css
+a {
+    padding-top: 1px;
+    padding-right: 1px;
+    padding-bottom: 1px;
+    padding-left: 1px; }
+/** ↑
+ *  These longhand properties */
+```
+
+The longhand properties in the example above can be more concisely written as:
+
+```css
+a {
+  padding: 1px;
+}
+```
+
+This rule warns when the following shorthand properties can be used:
+
+- `padding`
+- `margin`
+- `background`
+- `font`
+- `border`
+- `border-top`
+- `border-bottom`
+- `border-left`
+- `border-right`
+- `border-width`
+- `border-style`
+- `border-color`
+- `border-radius`
+- `transition`
+
```

Comment:
We only need the `border` shorthand property listed here don't we?

The `border-top`, `border-bottom`, `border-left`, `border-right`, `border-width`, `border-style`, `border-color`, and `border-radius` properties are _not_ shorthand properties so they shouldn't be listed in this list.